letsdrill.rb

```diff
@@ -1,13 +1,24 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+	if integer <= 100 && integer >= 90
```

Comment:
I think each check could do just one numeric comparison. This first one could just check if the input is over 90.